### DISCUSSION TITLE
iOS打包上传问题

### DISCUSSION THREAD

**Discussion by VinsonZwx** [Tencent/libpag#2484] [Q&A] [answered]
更新到xode16，打包上传报错，
![报错](https://github.com/user-attachments/assets/b122f223-e62a-4cf5-b1f6-68b441f2ccd2)
xcode配置已修改
![WechatIMG74](https://github.com/user-attachments/assets/d328f497-0e00-4104-9f6a-22593a5c35c1)

**Comment by VinsonZwx** [comment]
集成方式为 pod 'libpag', '4.2.41'

**Reply by kevingpqi123** [reply, marked as answer]
目前 4.2 版本已经停止维护，可以升级至 4.3 版本
